gen3_util/cli/__init__.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+from gen3_util.config import Config
+from gen3_util.common import print_formatted
+
+
+import click
+
+ENV_VARIABLE_PREFIX = 'GEN3_UTIL'
+
+
+def _common_options(self):
+    """Insert common commands into Group or Command."""
+    assert len(self.params) == 0
+    self.params.insert(0,
```

Comment:
I haven't encountered this pattern before --
- Is `self.params` a List?
- Can any additional click Options be added by following the same `insert(n+1...` pattern?